<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Student_InfoDialogClass</class>
 <widget class="QDialog" name="Student_InfoDialogClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1184</width>
    <height>625</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Student_InfoDialog</string>
  </property>
  <widget class="QWidget" name="tool" native="true">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>131</width>
     <height>641</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(84,92,100);</string>
   </property>
   <widget class="QToolButton" name="personTBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>120</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border:none; background-color: rgb(84,92,100); color: white;min-height:55px; font:14px ‘黑体’;padding-left:28px;
}

QToolButton:checked{background:rgb(67,74,80);}</string>
    </property>
    <property name="text">
     <string>个人中心</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Book/user.svg</normalon>
     </iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QToolButton" name="borrowTBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>60</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border:none; background-color: rgb(84,92,100); color: white;min-height:55px; font:14px ‘黑体’;padding-left:28px;
}

QToolButton:checked{background:rgb(67,74,80);}</string>
    </property>
    <property name="text">
     <string>借阅情况</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Book/calendar-alt.svg</normalon>
     </iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QToolButton" name="bookTBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border:none; background-color: rgb(84,92,100); color: white;min-height:55px; font:14px ‘黑体’;padding-left:28px;
}

QToolButton:checked{background:rgb(67,74,80);}</string>
    </property>
    <property name="text">
     <string>图书信息</string>
    </property>
    <property name="icon">
     <iconset resource="BookMS.qrc">
      <normaloff>:/Book/book-open-white.svg</normaloff>:/Book/book-open-white.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>16</width>
      <height>30</height>
     </size>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
   <widget class="QToolButton" name="recommendTBtn">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>180</y>
      <width>131</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QToolButton{border:none; background-color: rgb(84,92,100); color: white;min-height:55px; font:14px ‘黑体’;padding-left:28px;
}

QToolButton:checked{background:rgb(67,74,80);}</string>
    </property>
    <property name="text">
     <string>书籍推荐</string>
    </property>
    <property name="icon">
     <iconset>
      <normalon>:/Book/star (1).svg</normalon>
     </iconset>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
    <property name="toolButtonStyle">
     <enum>Qt::ToolButtonStyle::ToolButtonTextBesideIcon</enum>
    </property>
    <property name="autoRaise">
     <bool>true</bool>
    </property>
    <attribute name="buttonGroup">
     <string notr="true">buttonGroup</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QStackedWidget" name="stackedWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>0</y>
     <width>1011</width>
     <height>621</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="BookMS.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
